RecoEcal/EgammaClusterProducers/src/PFECALSuperClusterProducer.cc

```diff
@@ -356,6 +355,18 @@ void PFECALSuperClusterProducer::fillDescriptions(edm::ConfigurationDescriptions
   desc.add<double>("thresh_PFClusterBarrel", 0.0);
   desc.add<std::string>("PFBasicClusterCollectionBarrel", "particleFlowBasicClusterECALBarrel");
   desc.add<bool>("useRegression", true);
+
+  // Track isolation parameters: begin
+  desc.add<edm::InputTag>("trackProducer", edm::InputTag(""));
+  desc.add<double>("trkIsoPtMin", 0.5);
+  desc.add<double>("trkIsoConeSize", 0.4);
+  desc.add<double>("trkIsoZSpan", 999999.9);
+  desc.add<double>("trkIsoRSpan", 999999.9);
+  desc.add<double>("trkIsoVetoConeSize", 0.06);
+  desc.add<double>("trkIsoStripBarrel", 0.03);
+  desc.add<double>("trkIsoStripEndcap", 0.03);
```

Comment:
it would be better if this was its own seperate PSet containing the parameters as is down in the electron producer